Emby.Server.Implementations/Localization/LocalizationManager.cs

```diff
@@ -111,49 +99,55 @@ public IEnumerable<CultureDto> GetCultures()
 
         private async Task LoadCultures()
         {
-            List<CultureDto> list = new List<CultureDto>();
+            List<CultureDto> list = [];
 
-            await using var stream = _assembly.GetManifestResourceStream(CulturesPath)
-                ?? throw new InvalidOperationException($"Invalid resource path: '{CulturesPath}'");
-            using var reader = new StreamReader(stream);
-            await foreach (var line in reader.ReadAllLinesAsync().ConfigureAwait(false))
+            using var stream = _assembly.GetManifestResourceStream(CulturesPath);
+            if (stream is null)
             {
-                if (string.IsNullOrWhiteSpace(line))
-                {
-                    continue;
-                }
-
-                var parts = line.Split('|');
-
-                if (parts.Length == 5)
+                throw new InvalidOperationException($"Invalid resource path: '{CulturesPath}'");
+            }
+            else
+            {
+                using var reader = new StreamReader(stream);
+                await foreach (var line in reader.ReadAllLinesAsync().ConfigureAwait(false))
                 {
-                    string name = parts[3];
-                    if (string.IsNullOrWhiteSpace(name))
+                    if (string.IsNullOrWhiteSpace(line))
                     {
                         continue;
                     }
 
-                    string twoCharName = parts[2];
-                    if (string.IsNullOrWhiteSpace(twoCharName))
-                    {
-                        continue;
-                    }
+                    var parts = line.Split('|');
 
-                    string[] threeletterNames;
-                    if (string.IsNullOrWhiteSpace(parts[1]))
-                    {
-                        threeletterNames = new[] { parts[0] };
-                    }
-                    else
+                    if (parts.Length == 5)
```

Comment:
dont we rather want to throw here instead of silently ignoring malformatted values?